assistantv2/assistant_v2_test.go

```diff
@@ -15,31 +31,32 @@ var _ = Describe("AssistantV2", func() {
 	Describe("CreateSession(createSessionOptions *CreateSessionOptions)", func() {
 		createSessionPath := "/v2/assistants/{assistant_id}/sessions"
 		version := "exampleString"
+		accessToken := "0ui9876453"
```

Comment:
we should call these `bearerToken`

Reply:
This is something we will update with the automated unit tests

Reply:
I made this change in my generator PR.